src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -618,7 +649,13 @@ export default {
 		 */
 		async handleShouldRefreshConversations(options) {
 			if (options?.all === true) {
-				this.roomListModifiedBefore = 0
+				if (this.isCurrentTabLeader) {
+					this.roomListModifiedBefore = 0
```

Comment:
```suggestion
					this.roomListModifiedBefore = 0
					this.forceFullRoomListRefreshAfterXLoops = 10
```

follow-up: restart cycle